src/osip-monitoring-view/src/main/java/edu/kit/pse/osip/monitoring/view/dashboard/UIOutputStream.java

```diff
@@ -0,0 +1,207 @@
+package edu.kit.pse.osip.monitoring.view.dashboard;
+
+import java.io.PrintStream;
+import java.util.Locale;
+
+/**
+ * This class can be used to print messages of the default output and error output stream to a GUI console.
```

Comment:
https://github.com/ByteHamster/PSE/blob/master/docs/Pflichtenheft.tex#L272

Ich habe mich an dieser Stelle nach dem Pflichtenheft gerichtet und nach einem vertieften Blick ins Designdokument haben wir nur festgelegt, die Fehler in keine Datei zu schreiben. Damit ist das mit den Fehlern in die Konsole offen. Ich hätte es aber gelassen, wenn wir es jetzt schon haben.